From a3f157c97ba64d1e96dab705656ca5eadb251538 Mon Sep 17 00:00:00 2001
From: Rodrigo Moya <rodrigo@gnome-db.org>
Date: Tue, 20 Sep 2011 15:44:20 +0200
Subject: [PATCH 02/14] shell: Show the label for the category sections, which
 was being shown because of the gtk_widget_show_all
 replaced in previous commit

---
 shell/cc-shell-category-view.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/shell/cc-shell-category-view.c b/shell/cc-shell-category-view.c
index b2a5ee4..e52f558 100644
--- a/shell/cc-shell-category-view.c
+++ b/shell/cc-shell-category-view.c
@@ -164,6 +164,7 @@ cc_shell_category_view_constructed (GObject *object)
       gtk_label_set_attributes (GTK_LABEL (label), attrs);
       pango_attr_list_unref (attrs);
       gtk_frame_set_label_widget (GTK_FRAME (object), label);
+      gtk_widget_show (label);
 
       g_signal_connect (label, "style-set", G_CALLBACK (label_style_set_cb),
                         NULL);
-- 
1.7.6.1

